extra-dev/packages/coq-mi-cho-coq/coq-mi-cho-coq.dev/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+synopsis: "A specification of Michelson in Coq to prove properties about smart contracts in Tezos"
+maintainer: "raphael.cauderlier@nomadic-labs.com"
+authors: [ "Raphaël Cauderlier" "Bruno Bernardo" "Julien Tesson" "Arvid Jakobsson" ]
+
+homepage: "https://gitlab.com/nomadic-labs/mi-cho-coq/"
+dev-repo: "git+https://gitlab.com/nomadic-labs/mi-cho-coq/"
+bug-reports: "https://gitlab.com/nomadic-labs/mi-cho-coq/issues"
+license: "MIT"
+
+build: [
+  ["./configure"]
+  [make "-j%{jobs}%"]
+]
+install: [
+  make "install"
+]
+depends: [
+  "coq" {>= "8.8"}
+  "coq-ott"
+  "ott"
```

Comment:
```suggestion
  "ott" {build & >= "0.29"}
```
Only needed when building, not when the definitions have been generated (then only the Coq support lib is needed). Version constraint to match the Coq library.